lib/spec.gb

```diff
@@ -76,11 +76,21 @@ class Example
     @test_result = expectation.call(result)
   end
 
+  def not_to(expectation)
+    @test_result = !expectation.call(result)
+  end
+
   def eq(expectation)
     Block.new do |n|
       n == expectation
     end
   end
 
+  def ==(expectation)
```

Comment:
when will we use this method?

Reply:
yikes. I've just realized that this is an operator I used to use long ago (`should == value`).  
it is still included in RSpec, although in different form (`expect(x).to be == (y)`).

let me know what's your preference - I can remove that commit, then in the future could be implemented as `be ==` (if deemed appropriate).

Reply:
I think we can skip `==` for now. Cause the `should == value` is a deprecated pattern and I'm not plan to support it. And I don't think people would like to use `expect(x).to be == (y)` since we already have `eq`, which is more expressive in this case

Reply:
ok. I've removed it. I've also renamed the test suite, since it's not focused on the `Spec` class itself.